pkgs/games/endgame-singularity/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, fetchurl, unzip, python2Packages }:
+
+python2Packages.buildPythonApplication rec {
+  pname = "endgame-singularity";
+  version = "0.30c";
+  format = "other";
+
+  srcs =
+    [
+      (fetchurl {
+        url = "http://www.emhsoft.com/singularity/singularity-${version}-src.tar.gz";
+        sha256 = "13zjhf67gmla67nkfpxb01rxs8j9n4hs0s4n9lnnq4zgb709yxgl";
+      })
+      (fetchurl {
+        url = "http://www.emhsoft.com/singularity/endgame-singularity-music-007.zip";
+        sha256 = "0vf2qaf66jh56728pq1zbnw50yckjz6pf6c6qw6dl7vk60kkqnpb";
+      })
+    ];
+  sourceRoot = ".";
+
+  buildInputs = [ unzip ]; # The music is zipped
+  propagatedBuildInputs = with python2Packages; [ python pygame numpy ];
+
+  # Add shebang to main file
+  postPatch = ''
+    echo '#!${python2Packages.python}/bin/python'\
+         $'\n'$(cat singularity-${version}/singularity.py)\
+         > singularity-${version}/singularity.py
```

Comment:
Will this cause problems? Should I use `sponge`?

Reply:
I don't know `sponge`. Maybe it would be easier and nicer to just add a wrapper script `endgame-singularity` that just calls `${python2Packages.python}/bin/python $out/bin/singularity.py` or something?

Reply:
I managed to do it nicely with --add-flags